models/reservation.js

```diff
@@ -1,23 +1,19 @@
-const { object } = require('joi')
-const Joi = require('joi')
 const mongoose = require('mongoose')
-Joi.objectId = require('joi-objectid')(Joi)
 
 const reservationSchema = new mongoose.Schema(
   {
-    userId: {
+    user: {
```

Comment:
do we store userId here or user?

Reply:
it's better user, we I use populate from mongoose then you will have userId: { email: ... }, so it will be wrong because this is then not id but the whole user object